L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

```diff
@@ -728,23 +798,32 @@ bool l1t::TriggerMenuParser::parseScales(std::map<std::string, tmeventsetup::esS
 
     // Now the Pt LUTs  (??? more combinations needed ??)
     // ---------------
-    parsePt_LUTS(scaleMap, "EG",  precisions["PRECISION-EG-MU-MassPt"] );
-    parsePt_LUTS(scaleMap, "MU",  precisions["PRECISION-EG-MU-MassPt"] );
-    parsePt_LUTS(scaleMap, "JET", precisions["PRECISION-EG-JET-MassPt"] );
-    parsePt_LUTS(scaleMap, "TAU", precisions["PRECISION-EG-TAU-MassPt"] );
-   
+    parsePt_LUTS(scaleMap, "Mass" ,"EG",  precisions["PRECISION-EG-MU-MassPt"] );
```

Comment:
"PRECISION" stuff was introduced to cope with the precisions (digits after decimal point) for fixed-point computations appeared in [1]
These were defined for all the combinations just in case. In the end, all the precisions for "Pt" are 1. (as the current scales dictates)

* That said, I do not know if there is a need to use separate "Pt" precisions for "Mass" and "TwoBody" at all.
* Less confusing way to get precisions for "Pt" would be
  * precisions["PRECISION-EG-EG-MassPt"] for EG
  * precisions["PRECISION-MU-MU-MassPt"] for MU
  * precisions["PRECISION-JET-JET-MassPt"] for JET
  * precisions["PRECISION-TAU-TAU-MassPt"] for TAU

As we introduce transverse mass, don't we need precisions["PRECISION-ETM-ETM-MassPt"] and precisions["PRECISION-HTM-HTM-MassPt"] for missing-Et and missing-Ht, respectively ? (they should be all 1 though)

[1] https://svnweb.cern.ch/trac/cactus/browser/trunk/cactusprojects/utm/tmVhdlProducer/jinjaTemplates/gtl_pkg.vhd?rev=44313#L320